suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -568,21 +568,21 @@ export const isAccountOutdated = (account: Account, freshInfo: AccountInfo) => {
     // different sequence or balance
     const changedRipple =
         account.networkType === 'ripple' &&
-        (freshInfo.misc!.sequence !== account.misc.sequence ||
+        (freshInfo.misc?.sequence !== account.misc.sequence ||
             freshInfo.balance !== account.balance ||
-            freshInfo.misc!.reserve !== account.misc.reserve);
+            freshInfo.misc?.reserve !== account.misc.reserve);
 
     const changedEthereum =
-        account.networkType === 'ethereum' && freshInfo.misc!.nonce !== account.misc.nonce;
+        account.networkType === 'ethereum' && freshInfo.misc?.nonce !== account.misc.nonce;
 
     const changedCardano =
         account.networkType === 'cardano' &&
         // stake address (de)registration
-        (freshInfo.misc!.staking?.isActive !== account.misc.staking.isActive ||
+        (freshInfo.misc?.staking?.isActive !== account.misc.staking.isActive ||
             // changed rewards amount (rewards are distributed every epoch (5 days))
-            freshInfo.misc!.staking?.rewards !== account.misc.staking.rewards ||
+            freshInfo.misc?.staking?.rewards !== account.misc.staking.rewards ||
             // changed stake pool
-            freshInfo.misc!.staking?.poolId !== account.misc.staking.poolId);
+            freshInfo.misc?.staking?.poolId !== account.misc.staking.poolId);
```

Comment:
Hmm not sure what is worse - having non-null assertion here and risking `cannot read something of undefined` errors or changing it to optional chaining and risking false positives in return value of `isAccountOutdated` since for example `account.misc.nonce` will always be defined (see `getAccountSpecific`) and `freshInfo.misc?.nonce` might be undefined which could, in theory, mean the same as if fallback from `getAccountSpecific` is used?